src/main/java/com/svydovets/bibirnate/session/Session.java

```diff
@@ -1,5 +1,16 @@
 package com.svydovets.bibirnate.session;
 
+/**
+ * This is main interface to work with Bibirnate. It allows to read operations for mapped classes.
+ */
 public interface Session {
+    /**
+     * Find entity with provided entity type and identifier.
+     *
+     * @param id of entity
+     * @param type mapped class with {@link com.svydovets.bibirnate.annotation.Entity}
+     * @param <T>  type of entity
+     * @return found entity or null if no such entity was found
+     */
     <T> T findById(Object id, Class<T> type);
```

Comment:
What do you think if we didn't return null and return Optional instead? Or maybe this method should throw an exception if the entity is not present....

Reply:
Throwing is not an option because it would force you to use try/catch. I will update with optional.